src/main/java/EchoClient.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.net.Socket;
+
+public class EchoClient {
+    private ConsoleIO consoleIO;
+    private Socket echoSocket;
+
+    public static void main(String[] args) {
+        try {
+            String hostName = "localhost";
+            int portNumber = 8000;
```

Comment:
What if you want to run your echoServer on the port 8001? I think you would have to recompile and rerun your app
You can avoid that by writing that on a config file that your app would read when you run it 👍